src/pages/signin/ResendValidationForm.js

```diff
@@ -15,6 +16,11 @@ import compose from '../../libs/compose';
 const propTypes = {
     /* Onyx Props */
 
+    credentials: PropTypes.shape({
```

Comment:
Can you add `/** The credentials of the logged in person */` above so it matches the format in `src/pages/signin/ChangeExpensifyLoginLink.js`